inst/include/sampler.h

```diff
@@ -151,6 +151,16 @@ namespace smc {
         long GetNumber(void) const {return N;}
         ///Returns the number of evolution times stored in the history.
         long GetHistoryLength(void) const {return History.size();}
+        ///Returns the current particle set stored in the history.
+        population<Space> GetHistoryPopulation(long n) const {return History[n].GetValues();}
+        ///Returns a reference to the particle set stored in the history.
+        population<Space> & GetHistoryPopulationRefs(long n) {return History[n].GetValues();}
+        ///Returns the history flags
+        historyflags GetHistoryFlags(long n) {return History[n].GetFlags();}
+        ///Returns the history ###################
```

Comment:
Misleading comment: it returns the historical effective sample size...

Reply:
I have used this just as a placeholder :D.. I will write a good comment now.